app/forms/state_file/email_address_form.rb

```diff
@@ -3,6 +3,7 @@ class EmailAddressForm < QuestionsForm
     set_attributes_for :intake, :email_address
 
     validates :email_address, 'valid_email_2/email': true
+    validates :email_address, presence: true
```

Comment:
currently email address page on the login-options flow allows you to proceed without submitting anything